subsquid/src/server-extension/resolvers/pabloOverviewStats.ts

```diff
@@ -35,27 +44,34 @@ export class PabloOverviewStatsResolver
   constructor(private tx: () => Promise<EntityManager>) {}
 
   @FieldResolver({ name: "totalValueLocked", defaultValue: 0 })
-  async totalValueLocked(): Promise<bigint> {
+  async totalValueLocked(): Promise<TVL[]> {
     const manager = await this.tx();
 
-    let lockedValue: { amount: bigint }[] = await manager
-      .getRepository(HistoricalLockedValue)
-      .query(
-        `
-        SELECT
-          amount
-        FROM historical_locked_value
-        WHERE source = 'Pablo'
-        ORDER BY timestamp DESC
-        LIMIT 1
-      `
-      );
+    const lockedValue = await manager.find(HistoricalLockedValue, {
+      select: ["amount", "assetId"],
+      where: {
+        source: LockedSource.Pablo,
+      },
+    });
+
+    const totalValueLocked = lockedValue.reduce<Record<string, bigint>>(
+      (acc, value) => {
+        acc[value.assetId] = (acc[value.assetId] || 0n) + value.amount;
```

Comment:
Should this instead be?
```suggestion
        acc[value.assetId] = (acc[value.assetId] ?? 0n) + value.amount;
```

Reply:
It's the same in this case, as it is defaulting to 0 anyways. If this was `|| 1n`, it would matter as `0n || 1n = 1n`, but `0n ?? 1n = 0n`. Well noticed detail though 👍🏼